### PR TITLE
Edge Lightning Stuff

### PR DESCRIPTION
**Pull request by idoybh**

### CODE REVIEW

File: src/com/nitrogen/settings/fragments/NotificationSettings.java [hunk 1]
```diff
@@ -57,47 +54,14 @@ public void onCreate(Bundle icicle) {
...
-            int intHex = ColorPickerPreference.convertToColorInt(hex);
-            Settings.System.putInt(getContentResolver(),
-                    Settings.System.PULSE_AMBIENT_LIGHT_COLOR, intHex);
-            return true;
-        } else if (preference == mEdgeLightDurationPreference) {
-            int value = (Integer) newValue;
-            Settings.System.putInt(getContentResolver(),
-                    Settings.System.PULSE_AMBIENT_LIGHT_DURATION, value);
+        if (preference == mFlashOnCallWaitingDelay) {
+            int val = (Integer) newValue;
```
Comment (poad42) [inline src/com/nitrogen/settings/fragments/NotificationSettings.java]: don't have this in our stack

Comment (poad42) [reply src/com/nitrogen/settings/fragments/NotificationSettings.java]: I'll edit tho

Comment (idoybh) [reply src/com/nitrogen/settings/fragments/NotificationSettings.java]: woops. you can edit here btw

File: src/com/nitrogen/settings/fragments/NotificationSettings.java [hunk 2]
```diff
@@ -76,20 +78,38 @@ public void onCreate(Bundle icicle) {
...
 
     @Override
     public boolean onPreferenceChange(Preference preference, Object newValue) {
-        if (preference == mEdgeLightColorPreference) {
-            String hex = ColorPickerPreference.convertToRGB(
+        if (preference == mFlashOnCallWaitingDelay) {
+            int val = (Integer) newValue;
+            Settings.System.putInt(getContentResolver(), Settings.System.FLASH_ON_CALLWAITING_DELAY, val);
+            return true;
+        } else if (preference == mEdgeLightColorPreference) {
```
Comment (idoybh) [inline src/com/nitrogen/settings/fragments/NotificationSettings.java]:
```suggestion
        if (preference == mEdgeLightColorPreference) {
```

**Comment by idoybh** [comment]
Done. Just rebase and add as a fixup after merge

File: res/values/nitrogen_strings.xml [hunk 3]
```diff
@@ -232,14 +232,6 @@
     <string name="media_art_title">Media cover art</string>
     <string name="media_art_summary">Enable media cover art on the lockscreen</string>
 
-    <!-- Ambient edge lighting -->
-    <string name="notification_screen_title">Screen</string>
```
Comment (poad42) [inline res/values/nitrogen_strings.xml:236]: string is needed for pref category title

**Comment by poad42** [comment]
https://github.com/op5-q/android_packages_apps_NitrogenSettings/commits/ten-edge

pushed here for now, preview works but uh doesn't work in ambient donno why log shows ambientlight is enabled,

frameworks/base pushed here: https://github.com/op5-q/frameworks_base/tree/ten-edge

**Comment by idoybh** [comment]
Wdym doesn't work in ambient? Gotta be fwb side

**Comment by idoybh** [comment]
@poad42 force pushed you fix. Amended to the right commits

**Pull request merged**